### ISSUE TITLE
Opening links in new tab points to not existing /TendrlModule/ page

### ISSUE THREAD

**Issue by dahorak** [Tendrl/ui#868]
It is not possible to open any link in new tab (using *Ctrl+left click* or *middle button click*, because it leads to not existing URL containing. For example:
`http://tendrl-server.example.com/TendrlModule/#/cluster-volumes/9d89ec2a-38f2-4a3a-8282-d63fc08b7809
`
Instead of:
`
http://tendrl-server.example.com/#/cluster-volumes/9d89ec2a-38f2-4a3a-8282-d63fc08b7809
`
And return *404 Not Found* page:
```
Not Found
The requested URL /TendrlModule/ was not found on this server.
```
-----

The root cause of this issue seems to be following tag in the `<head>...</head>` section (https://github.com/Tendrl/ui/blob/master/src/index.html#L17):
`<base href="/TendrlModule/">`

If you will try to remove it (on the fly using *Firefox Developer Tools*), it seems to work as expected.
What is the reason to explicitly configure *base href* to `/TendrlModule/`?

**Comment by dahorak** [comment]
See also https://github.com/Tendrl/ui/issues/577

**Comment by gnehapk** [comment]
@dahorak Yes, it works after removing `/TendrlModule/`. Thanks. Will send the patch in my next PR.

**Comment by gnehapk** [comment]
@dahorak  Fixed in tendrl-ui-1.6.1-20180317T042235.84aac52.noarch.rpm build. Please verify and close the issue.

**Comment by dahorak** [comment]
I can confirm, that it works with the latest *master* builds.
Tested with `tendrl-ui-1.6.1-20180319T082533.cab4a4e.noarch`.
